styles.css

```diff
@@ -1,382 +1,495 @@
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap: 88px;
+  background-color: white;
+  border-top-right-radius: 6rem;
+  height: 1280px;
+  margin-top: 5rem;
+  padding: 114px 24px;
+}
+
+.about__container h1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #172b4d;
+}
+
+.about__container p {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+.let__g {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #7f8cff;
+}
+
+.social__media__lin {
+  list-style: none;
+  margin-right: 0.9rem;
+  margin-top: 0.5rem;
+}
+
+.social__media__imag {
+  width: 24px;
+  height: 14px;
+  border-radius: 8px;
+}
+
+.resume_btn {
+  width: 162px;
+  height: 48px;
+  padding: 12px;
+  border-radius: 8px;
+  background-color: transparent;
+  border: 1px solid #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+}
+
+.lang__header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.lang__header h1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.lang__container {
+  display: grid;
+  gap: 1rem;
+  grid-template-columns: 1fr;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.lang__container__type {
+  display: flex;
+  gap: 1rem;
+  padding-left: 1rem;
+  height: 66px;
+  background-color: #f7f7f9;
+  align-items: center;
+  border-radius: 8px;
+}
+
+.lang__container__type img {
+  width: 48px;
+  height: 48px;
+}
+
+.lang__container__type span {
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.framework {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 2rem;
+  padding: 6px;
+  border-bottom: 3px solid #f7f7f9;
+}
+
+.framework img {
+  margin-right: 1rem;
+  color: #f7f7f9;
+}
+
+.social_media {
+  margin: 0;
+  display: flex;
+  padding: 1rem 0 1rem 0;
+}
+
+/* ======================CONTACT ME========================== */
+
+.contact__container {
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-top: 5rem;
+  position: relative;
+  height: 779px;
+  bottom: 159px;
+}
+
+.contact__container .backgrnd {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(img/contact\ form\ shapes-mobile@2x.svg);
+  background-position: top 48px right;
+  height: 140vh;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  border-top-left-radius: 6rem;
+  width: 100%;
+}

```

Comment:
- [ ] As mentioned previously the background image at the bottom of the contact section is not aligned properly I noticed that it has now been completely removed and this causes a discrepancy when compared with the Figma design so please make sure to add it and also make it sure it's aligned in the proper position. Also please make sure to remove the extra space at the bottom.
- Figma design:
  ![image](https://github.com/sibomana123/MV__Portfolio/assets/104968236/00f96a80-723c-4470-bfef-f331f98ff8c1)
- Your design:
  ![image](https://github.com/sibomana123/MV__Portfolio/assets/104968236/d9d94d64-2089-407e-a717-e457929aa471)